test/hotspot/jtreg/compiler/print/TestPrintInliningLateVirtualCall.java

```diff
@@ -44,9 +44,9 @@ public static void main(String[] args) {
     private static void testHelper(int i) {
         A a;
         if (i == 10) {
-            a = fieldA;
-        } else if (i > 10) {
             a = fieldB;
+        } else if (i > 10) {
+            a = fieldA;
```

Comment:
Why do you need these changes?

Reply:
It was so I could verify the change I made to the late virtual call message. Otherwise, the change in the c2 code doesn't cause any difference in the `PrintInlining` output.

Reply:
Ah, got it. Makes sense.